apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -33,278 +34,263 @@ export function Chat({
+          )}].  Then answer this question: ${userContent}`,
+        },
+    ).then((response) => {
+      // * Loading: Here is the information you need... 'finish'
+      setLoadingState("finished");
+      // scrollToBottom();
+
+      return response;
+    });
+  };
 
   useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: activeThread

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)